src/main/java/org/breedinginsight/services/writers/ExcelWriter.java

```diff
@@ -86,8 +100,8 @@ public static InputStream writeToInputStream(String sheetName, List<Column> colu
     }
 
     //Writes doc to file in project, for optional testing
-    public static void writeToFile(String fileName, String sheetName, List<Column> columns, List<Map<String, Object>> data) throws IOException {
-        XSSFWorkbook workbook = writeToWorkbook(sheetName, columns, data);
+    public static void writeToFile(String fileName, String sheetName, List<Column> columns, List<Map<String, Object>> data, FileType extension) throws IOException {
+        Workbook workbook = writeToWorkbook(sheetName, columns, data, extension);
 
         try (FileOutputStream fileOutput = new FileOutputStream(fileName +".xlsx")) {
```

Comment:
Shouldn't this line be updated to use the extension value from the `extension` enum?

Reply:
good catch, fixed